src/web/mjs/connectors/BilibiliManhua.mjs

```diff
@@ -23,26 +23,83 @@ export default class BilibiliManhua extends Connector {
         };
         //default config doesnt need language switch, its fixed to CN
         this.config = {
-            quality:  {
-                label: 'Preferred format',
-                description: 'format of images\nwebp (low)\njpg (medium)\npng (high))',
-                input: 'select',
-                options: [
-                    { value: 'webp', name: 'webp' },
-                    { value: 'jpg', name: 'jpg' },
-                    { value: 'png', name: 'png' },
+            format: {
+                label:'Preferred format',
+                description:'Format of images\nwebp(low)\n jpg(medium)\n png(high))',
+                input:'select',
+                options:[
+                    { value:'webp', name:'webp' },
+                    { value:'jpg', name:'jpg' },
+                    { value:'png', name:'png' },
                 ],
                 value: 'png'
             },
             language: {
                 label: 'Language Settings',
-                description: 'Choose the language to use. This will affect available manga lists',
+                description: 'Choose the language to use. This will affect available manga in list',
                 input: 'disabled',
-                value : 'cn'
+                value: 'cn'
+            },
+            picquality: {
```

Comment:
This settings seems redundant, users always ask for the best possible image quality and resolution.

Reply:
Same answer